torch_lr_finder/lr_finder.py

```diff
@@ -460,6 +463,19 @@ def plot(self, skip_start=10, skip_end=5, log_lr=True, show_lr=None, ax=None):
 
         # Plot loss as a function of the learning rate
         ax.plot(lrs, losses)
+
+        # Plot the suggested LR in a one-time loop
+        while suggestion:
```

Comment:
I don't see a reason for the loop. Looks like the `while` can just be replaced with an `if`

Reply:
```suggestion
        # Plot the suggested LR in a one-time loop
        if suggest_lr:
```